app/models/submission.rb

```diff
@@ -70,6 +70,8 @@ class Submission < ApplicationRecord
   scope :completed, -> { where.not(state: 'draft') }
   scope :draft, -> { where(state: 'draft') }
   scope :submitted, -> { where(state: 'submitted') }
+  scope :available,
```

Comment:
Mind briefly explaining what a `scope` is? (I will also read docs, but i think in your own words TLDR explainations are also super useful for concepts.)

Reply:
Sure! It's a convenient way to specify common queries. It's a shortcut for defining a method - check it out:

```ruby
class Submission < ApplicationRecord::Base
  scope(:available) -> { where(state: APPROVED, consigned_partner_submission_id: nil) }

  # that's a shortcut for doing this...

  def self.available
    where(state: APPROVED, consigned_partner_submission_id: nil)
  end
end
```

Those two are the same. If that Ruby syntax is weird, think of it as two arguments to a method called `scope` - the first is a symbol that is the name of a method we're going to define. The second argument (sometimes called a [stabby lambda](https://www.honeybadger.io/blog/using-lambdas-in-ruby/)) is what's evaluated when the method is called.

This is a common enough pattern that Rails gives us a helper to make these things a bit easier and then it's nice because you can search a file for `scope` and get a sense of the important queries for a given model.

Here's the guides section on it:

https://guides.rubyonrails.org/active_record_querying.html#scopes